python_modules/libraries/dagster-dg/dagster_dg/scaffold.py

```diff
@@ -285,5 +288,10 @@ def scaffold_library_object(
         str(path),
         *(["--json-params", json.dumps(scaffold_params)] if scaffold_params else []),
         *(["--scaffold-format", scaffold_format]),
+        *(
+            ["--project-root", str(dg_context.root_path)]
+            if dg_context.dagster_version > MIN_DAGSTER_SCAFFOLD_PROJECT_ROOT_OPTION_VERSION
```

Comment:
oof this is a scary indication of things to come in terms dealing with version skew